docs/source/docs/level-up/intermediate_skills/10_data_generation.rst

```diff
@@ -0,0 +1,43 @@
+===========================
+Level 10: Data Generation
+===========================
+
+
+Pre-training of deep learning models for vision tasks can increase model accuracy.
+Training model with the synthetic dataset is one of famouse pre-training approach
+since the manual annotations is quite expensive work.
+
+Base on the `previous research <https://arxiv.org/abs/2103.13023>`_,
```

Comment:
Please use citation.
https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#citations

Reply:
Cool. I modified to use citation.